Subject: Add support for another AX8817x network device,
From: olh@suse.de
Patch-mainline: almost

Add support for another AX8817x network device,
found in a docking station.


T:  Bus=01 Lev=02 Prnt=03 Port=00 Cnt=01 Dev#=  4 Spd=480 MxCh= 0
D:  Ver= 2.00 Cls=02(comm.) Sub=00 Prot=00 MxPS=64 #Cfgs=  1
P:  Vendor=1631 ProdID=6200 Rev= 0.01
S:  Manufacturer=goodway corp usb
S:  Product=gwusb2e
S:  SerialNumber=01
C:* #Ifs= 1 Cfg#= 1 Atr=80 MxPwr=450mA
I:  If#= 0 Alt= 0 #EPs= 3 Cls=00(>ifc ) Sub=00 Prot=00 Driver=(none)
E:  Ad=81(I) Atr=03(Int.) MxPS=   8 Ivl=128ms
E:  Ad=02(O) Atr=02(Bulk) MxPS= 512 Ivl=0ms
E:  Ad=83(I) Atr=02(Bulk) MxPS= 512 Ivl=0ms


Signed-off-by: Olaf Hering <olh@suse.de>

diff -purN ../linux-2.6.10.orig/drivers/usb/net/usbnet.c ./drivers/usb/net/usbnet.c
--- ../linux-2.6.10.orig/drivers/usb/net/usbnet.c	2004-12-24 21:34:58.000000000 +0000
+++ ./drivers/usb/net/usbnet.c	2005-01-19 20:18:17.399020430 +0000
@@ -3265,6 +3265,10 @@ static const struct usb_device_id	produc
 	// Surecom EP-1427X-2
 	USB_DEVICE (0x1189, 0x0893),
 	.driver_info = (unsigned long) &ax8817x_info,
+}, {
+	// goodway corp usb gwusb2e
+	USB_DEVICE (0x1631, 0x6200),
+	.driver_info = (unsigned long) &ax8817x_info,
 },
 #endif
 


-------------------------------------------------------
This SF.Net email is sponsored by: IntelliVIEW -- Interactive Reporting
Tool for open source databases. Create drag-&-drop reports. Save time
by over 75%! Publish reports on the web. Export to DOC, XLS, RTF, etc.
Download a FREE copy at http://www.intelliview.com/go/osdn_nl
_______________________________________________
linux-usb-devel@lists.sourceforge.net
To unsubscribe, use the last form field at:
https://lists.sourceforge.net/lists/listinfo/linux-usb-devel

